content/md/first-steps/manage-your-attributes.md

```diff
@@ -7,77 +7,147 @@ ee-only: false
 related: what-is-an-attribute
 ---
 
-# Overview
+If you are not familiar with the concept of attributes, you can read our [What is an attribute?](/articles/what-is-an-attribute.html#mainContent) article.
 
-To fit the PIM application to your needs, Akeneo enables you to define attributes. Akeneo offers you 14 types of attributes on the Community Edition version, and 15 types on the Enterprise Edition version.
+# See your attributes
 
-::: info
-The Asset Collection attribute is specific to the Enterprise Edition version. Learn more about it by discovering our [Product Asset Manager feature](/articles/work-with-assets.html).
-:::
+Go to `Settings/Attributes`, to see all the existing attributes or create a new one. 
 
-You will be able to select the most appropriate attribute type for your needs (see [Create an attribute](#create-an-attribute)). For example, you can choose to create a field as a text, in which users can write what they want, or as a drop down list (simple or multi select) with a selection of values, forcing users to select predefined options or add a `metric` attribute to fill in the weight/size/length... of your products.
+## Attribute grid pagination
 
-::: tips
-For `metric` attributes, you will be even offered the possibility to automatically convert metric values into others to fit your export needs see the [Manage your Channels](/articles/manage-your-channels.html#create-a-channel) article.
+The attribute grid displays 25 attributes per page. To browse attribute's pages, click on the page number you want to reach or use the filters.
+
+![image](../img/Settings_Attributes.png)    
+
+You can sort out attributes, by clicking on the column header on the grid.
+
+## Search on a attribute's label
+
+To quickly find an attribute, use the search bar available above the grid. Use your attribute's label to find what you are looking for.
+
+![image](../img/Settings-UsetheSearchBar.gif)
+
+::: warning
+If your attribute does not have any label translated yet, it will not pop in the search results. **Make sure your attribute has its labels translated.**
```

Comment:
`does not have yet any translated label`

Reply:
I didn't know where to place it so I've removed the yet :)